src/components/pages/meeting-rooms/index.tsx

```
@@ -1,18 +1,16 @@
 import { Resource } from "@/lib/api/amplify/helper";
 import { getGroupedResourceListBySubtype } from "@/lib/api/amplify/resource/utils";
 import { isOpenDrawerAtom } from "@/store/isOpenDrawerAtom";
 import { useQueryClient } from "@tanstack/react-query";
 import { AnimatePresence } from "framer-motion";
 import { useAtom, useAtomValue, useSetAtom } from "jotai";
-import { useMemo } from "react";
+import { Suspense, useMemo } from "react";
 
-// import { Suspense, useMemo } from "react";
-// import Error from "../seats/Error";
+import Error from "../seats/Error";
```

Biome

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)